src/__tests__/hooks/useFetchCompareResults.test.ts

```diff
@@ -23,7 +23,7 @@ describe('Tests useFetchCompareResults', () => {
       },
     } = renderHook(() => useFetchCompareResults(), { wrapper: StoreProvider });
     const spyOnFetch = jest.spyOn(global, 'fetch');
-    await dispatchFetchCompareResults(['fenix'], ['testRev'], '1');
+    dispatchFetchCompareResults(['fenix'], ['testRev'], '1');
```

Comment:
and I removed some useless `await`. I kept the `async` on the function because we might need it in the future so I thought this would be easier like this.